dispatch/dispatch.go

```diff
@@ -362,6 +364,10 @@ func (ag *aggrGroup) fingerprint() model.Fingerprint {
 	return ag.labels.Fingerprint()
 }
 
+func (ag *aggrGroup) GroupKey() uint64 {
+	return uint64(ag.labels.Fingerprint() ^ ag.routeFP)
```

Comment:
Not sticking to the "Fingerprint" type is fine by me as well as long as we stick with the same type. But in the context of this change I guess the way it is is fine then. Nevermind me here :)